test/extended/cli/mustgather.go

```diff
@@ -340,6 +344,294 @@ var _ = g.Describe("[sig-cli] oc adm must-gather", func() {
+		g.By("Set namespace as privileged namespace")
+		err := oc.AsAdmin().Run("label").Args("namespace", oc.Namespace(), "pod-security.kubernetes.io/enforce=privileged", "pod-security.kubernetes.io/audit=privileged", "pod-security.kubernetes.io/warn=privileged", "security.openshift.io/scc.podSecurityLabelSync=false", "--overwrite").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		err = oc.AsAdmin().Run("adm").Args("policy", "add-cluster-role-to-user", "cluster-admin", "system:serviceaccount:"+oc.Namespace()+":default").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		defer exec.Command("bash", "-c", "rm -rf /tmp/must-gather-56929").Output()
+		var wg sync.WaitGroup
+		wg.Add(1)
+		go func() {
+			defer g.GinkgoRecover()
+			defer wg.Done()
+			_, err = oc.AsAdmin().WithoutNamespace().Run("adm").Args("--run-namespace", oc.Namespace(), "must-gather", "--source-dir=/must-gather/static-pods/", "--dest-dir=/tmp/must-gather-56929").Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+		}()
+		err = wait.Poll(5*time.Second, 60*time.Second, func() (bool, error) {
+			output, err1 := oc.AsAdmin().Run("get").Args("pod", "-n", oc.Namespace(), "-l", "app=must-gather", "-o=jsonpath={.items[0].status.phase}").Output()
+			if err1 != nil {
+				e2e.Logf("the err:%v, and try next round", err1)
+				return false, nil
+			}
+			if matched, _ := regexp.MatchString("Running", output); matched {
+				e2e.Logf("Check the must-gather pod running in own namespace\n")
+				return true, nil
+			}
+			return false, nil
+		})
+		if err != nil {
+			oc.Run("get").Args("events", "-n", oc.Namespace()).Execute()
+		}
+		o.Expect(err).NotTo(o.HaveOccurred(), "Cannot find the must-gather pod in own namespace")
+		wg.Wait()
+		e2e.Logf("Must-gather command completed, starting pod cleanup check")
+		err = wait.Poll(10*time.Second, 600*time.Second, func() (bool, error) {
+			output, err1 := oc.AsAdmin().Run("get").Args("pod", "-n", oc.Namespace(), "-l", "app=must-gather").Output()
+			if err1 != nil {
+				e2e.Logf("the err:%v, and try next round", err1)
+				return false, nil
+			}
+			if matched, _ := regexp.MatchString("No resources found", output); matched {
+				e2e.Logf("Check the must-gather pod disappeared in own namespace\n")
+				return true, nil
+			}
+			return false, nil
+		})
+		o.Expect(err).NotTo(o.HaveOccurred(), "Still find the must-gather pod in own namespace even wait for 10 mins")
+	})
+
+	// author: yinzhou@redhat.com
+	g.It("Fetch audit logs of login attempts via oc commands timeout [apigroup:config.openshift.io][Timeout:20m]", func() {
+		controlPlaneTopology, err := exutil.GetControlPlaneTopology(oc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		// Skip on hypershift (External topology)
+		if *controlPlaneTopology == configv1.ExternalTopologyMode {
+			g.Skip("Hypershift clusters don't have oauth-server audit logs in must-gather")
+		}
+
+		g.By("run the must-gather")
+		tempDir, err := ioutil.TempDir("", "test.oc-adm-must-gather-51697.")
```

Comment:
```suggestion
		tempDir, err := ioutil.TempDir("", "test.oc-adm-must-gather.")
```
no need for the number here